/* ---EJERCICIOS: Manejo de errores--- */

// 1: Captura una excepcion utilizando try-catch:



// 2: Captura una excepcion utilizando try-catch y finally:



// 3: Lanza una excepcion generica:



// 4: Crea una excepcion personalizada:



// 5: Lanza una excepcion personalizada:



// 6: Lanza varias excepciones segun una logica definida:



// 7: Captura varias excepciones en un mismo catch:



// 8: Crea un bucle que intente transformar a float cada valor y capture y muestre los errores:



// 9: Crea una funcion que verifique si un objeto tiene una propiedad especifica y lance una excepcion personal:



// 10: Crea una funcion que realice reintentos en caso de error hasta un maximo de 10:

